src/Compilers/CSharp/Test/Semantic/Semantics/StackAllocInitializerTests.cs

```diff
@@ -58,9 +58,6 @@ unsafe void M()
                 // (10,29): error CS0208: Cannot take the address of, get the size of, or declare a pointer to a managed type ('RefG<string>')
                 //         var x2 = stackalloc RefG<string>[10];
                 Diagnostic(ErrorCode.ERR_ManagedAddr, "RefG<string>").WithArguments("RefG<string>").WithLocation(10, 29),
-                // (11,29): error CS0208: Cannot take the address of, get the size of, or declare a pointer to a managed type ('RefG<int>')
```

Comment:
FYI @dotnet/roslyn-compiler this is an expected baseline change, given that `master` added support for unmanaged generic types.
Relates to https://github.com/dotnet/roslyn/pull/33951